xtend-contrib/build.gradle

```diff
@@ -1,20 +1,24 @@
 plugins {
-	id 'org.xtend.xtend' version '0.4.5'
-	id 'com.github.oehme.sobula.bintray-release' version '0.4.1'
-	id 'com.github.oehme.sobula.stefan' version '0.4.1'
+	id 'org.xtext.xtend' version '1.0.3'
+	id 'com.github.oehme.sobula.bintray-release' version '0.6.6'
+	id 'com.github.oehme.sobula.stefan' version '0.6.6'
 }
 
 group = 'com.github.oehme.xtend'
 description = 'Automates well known Java Patterns using Xtend Active Annotations'
 repositories.jcenter()
 
+sourceSets {
+	main.xtendOutputDir = 'src/main/xtend-gen'
```

Comment:
.gitignore already contains xtend-gen/ (line 8) this applies to all directories called xtend-gen no matter where they are located. I was actually able to remove that line alltogether, it only used to be a problem with the old xtend-gradle-plugin. 

Added a comment on settings.gradle line setting the rootProject.name